lib/background_worker/base.rb

```diff
@@ -66,7 +82,9 @@ def perform_later(options = {})
         opts = options.symbolize_keys
         worker = new(options)
         # Enqueue to the background queue
+        worker.before_enqueue
         BackgroundWorker.enqueue(self, opts.merge(job_id: worker.job_id))
```

Comment:
We should change this to enqueue with the workers current options; as what we want to do with the before_enqueue in QuickTravel is to automatically add the "multi-tenancy" params to each job with a before_enqueue callback.

You may need to expose worker.options if its not already.